Projects/Feature/OnboardingFeature/Sources/Onboarding/OnboardingView.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  OnboardingView.swift
+//  OnboardingFeature
+//
+//  Created by 김지현 on 8/9/24.
+//  Copyright © 2024 PomoNyang. All rights reserved.
+//
+
+import SwiftUI
+import DesignSystem
+
+import ComposableArchitecture
+
+public struct OnboardingView: View {
+  @Bindable var store: StoreOf<OnboardingCore>
+
+  public init(store: StoreOf<OnboardingCore>) {
+    self.store = store
+  }
+
+  public var body: some View {
+    ZStack {
+      Alias.Color.Background.primary
+        .ignoresSafeArea()
+      VStack {
+        Spacer()
+        VStack(spacing: 0) {
+          TabView(selection: $store.currentItemID) {
+            ForEach(store.fakedData, id: \.hashValue) { item in
```

Comment:
이미 OnboardingItem이 Identifiable을 채택하고 있어서 이건 제거해도 괜찮을거 같아

```suggestion
            ForEach(store.fakedData) { item in
```